pkg/export/otel/metrics.go

```diff
@@ -536,6 +544,21 @@ func (mr *MetricsReporter) setupSpanMeters(m *Metrics, meter instrument.Meter) e
 	return nil
 }
 
+func (mr *MetricsReporter) setupHostInfoMeter(meter instrument.Meter) error {
+	if !mr.cfg.SpanMetricsEnabled() || !mr.cfg.ServiceGraphMetricsEnabled() {
```

Comment:
This check is duplicate before invoking setupHostInfoMeter, but also is inconsistent. Shouldn't be:
```
if !mr.cfg.SpanMetricsEnabled() && !mr.cfg.ServiceGraphMetricsEnabled() {
```
?

Reply:
Yeah, you are right, I'll remove it.